.circleci/config.yml

```diff
@@ -93,16 +110,41 @@ jobs:
       - store_artifacts:
           path: ./logs
 
+  deploy:
+    <<: *deploy_config
+    steps:
+      - *restore_repo
+      - *restore_mvn
+      - run: lein deploy clojars
+
 workflows:
   version: 2
-  build_and_deploy:
+  build_and_test:
     jobs:
       - checkout_code
       - deps:
           context: org-global
           requires:
             - checkout_code
-      - build:
+      - test:
           context: org-global
           requires:
             - deps
+
+  deploy:
+    jobs:
+      - checkout_tags:
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
+      - deploy:
+          context: org-global
+          requires:
+            - checkout_tags
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
I would prefer to keep the "v" out of the tag. Can we release on tags matching `x.y.z`?  (For instance, anything matching `^\d+\.\d+\.\d+$`.)

Reply:
Sure. But it was useful while testing to match version strings with arbitrary suffixes. Otherwise in order to test this workflow, you need to make an official looking release.

I think it would be useful to match `/^\d+\.\d+\.\d+.-/` to support this use-case and also allow for release candidates like "1.2.3-rc1"

Reply:
Should we exclude `SNAPSHOT` from the list of allowed suffixes?